test/organization.js

```diff
@@ -0,0 +1,119 @@
+const { expect } = require("chai");
+const chai = require("chai");
+const chaiHttp = require("chai-http")
+const server = "http://localhost:3000"
+const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjI1LCJyb2xlSWQiOjEsImlhdCI6MTY1MDkyMjIyNCwiZXhwIjoxNjUxMDA4NjI0fQ.cXzKVKxt2DXPkfy9KTjSBcuBLDH8MktEEhF1bYi2YpM"
+
+chai.should()
+chai.use(chaiHttp);
+
+
+describe("Task ONG", ()=>{
+
+  describe("Get /organization/public", ()=>{
+
+    it("It should GET organization data", (done)=>{
```

Comment:
No hace falta agergar el It al texto, para eso es la funcion it. Lo mismo para el resto

Reply:
ok